lib/stat_tracker.rb

```diff
@@ -23,6 +23,48 @@ def initialize(files)
     end
   end
 
+  #array off all gameteam objects played in that season
+  #can't get from just @games because doesn't have tackles so need to relate
+  #season_id to game id then game_id in gameteams data to game_id there
+  #this can definitely be refactored
+  def games_played_in_season(season)
+    game_objects_by_season = @games.group_by do |game|
+      game.season
```

Comment:
group_by is a great method to use here to pull out game objects by a season input. Correct me if I'm wrong, the structure would look something like this {"season" => [game_object1, game_object2, etc] }.

Reply:
That's right with the structure. But something tells me we could combine those first to iterations to only return an array with the games that are in that season? But it's nice to eventually have a hash with the single key/value pair using the .keep_if. Overall I think this could be parsed down as it's doing quite a few things to give us the return value. 

for the parameter, are you saying rename for all our methods? That makes sense to me and I'm down, just wanted to clarify.